cmake/globalutils.cmake

```diff
@@ -953,3 +953,14 @@ function(ivw_copy_if retval)
     set(${retval} ${res} PARENT_SCOPE)
 endfunction()
 
+function(ivw_deploy_qt target)
+    if(WIN32)
+        find_program(WINDEPLOYQT_EXECUTABLE NAMES windeployqt HINTS ${QTDIR} ENV QTDIR PATH_SUFFIXES bin)
+
+        get_filename_component(qt_bin_dir ${WINDEPLOYQT_EXECUTABLE} DIRECTORY  )
+        add_custom_command(TARGET ${target} POST_BUILD 
```

Comment:
Can't this be executed when running cmake instead?

Reply:
No, the `WINDEPLOYQT_EXECUTABLE` reads the built exe/dll to know what to copy, hence it needs to be `POST_BUILD`